acm/odh-edge/apps/bike-rental-app/bike-rental-inference-generator-cronjob.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: batch/v1
```

Comment:
If you expect this file to get deployed by default, I think you'll need to add it to the `resources` list in the  `kustomization.yaml` in the same folder

Reply:
Thanks!, was on my TO DO list indeed. Done.